### PR TITLE
Fix: errors for generic join with branch parameters were shown on the top of node

### PR DESCRIPTION
**Pull request by arkadius**

### CODE REVIEW

File: ui/client/components/graph/node-modal/NodeDetailsContent.js [hunk 1]
```diff
@@ -652,7 +652,11 @@ export class NodeDetailsContent extends React.Component {
 
   availableFields = () => {
     if (this.props.dynamicParameterDefinitions) {
-      return this.props.dynamicParameterDefinitions.map(param => param.name)
+      const commonFields = ["id", "outputVar"]
+      const paramFields = this.props.dynamicParameterDefinitions.map(param => param.name)
+      const branchParamsFields = this.state?.editedNode?.branchParameters
+        .flatMap(branchParam => branchParam.parameters.map(param => branchErrorFieldName(param.name, branchParam.branchId)))
+      _.concat(commonFields, paramFields, branchParamsFields)
```
Comment (mproch) [inline ui/client/components/graph/node-modal/NodeDetailsContent.js:659]: Don't you miss return here?

File: ui/client/components/graph/node-modal/NodeDetailsContent.js [hunk 2]
```diff
@@ -652,7 +652,11 @@ export class NodeDetailsContent extends React.Component {
 
   availableFields = () => {
     if (this.props.dynamicParameterDefinitions) {
-      return this.props.dynamicParameterDefinitions.map(param => param.name)
+      const commonFields = ["id", "outputVar"]
```
Comment (mproch) [inline ui/client/components/graph/node-modal/NodeDetailsContent.js:655]: can we extract common method with Join handling below?

Comment (arkadius) [reply ui/client/components/graph/node-modal/NodeDetailsContent.js]: good point, extracted

File: ui/client/components/graph/node-modal/NodeDetailsContent.js (hunk 1 above)
Comment (arkadius) [reply ui/client/components/graph/node-modal/NodeDetailsContent.js]: good catch, added return

**Review by mproch — APPROVED** [review]

**Pull request merged**